protostar/migrator/migrator_contract_identifier_resolver.py

```diff
@@ -1,32 +1,48 @@
 from pathlib import Path
 
-from protostar.compiler import CompiledContractWriter, ProjectCompiler
+from protostar.compiler import ProjectCompiler
 from protostar.migrator.migrator_datetime_state import MigratorDateTimeState
+from protostar.protostar_exception import ProtostarException
+
+
+class ContractIdentificationException(ProtostarException):
+    pass
+
+
+def checked_contract_path(contract_path: Path) -> Path:
```

Comment:
```suggestion
def check_contract_path(contract_path: Path) -> Path:
```
[Consistency, Style] Methods and functions names usually start with a verb.

Reply:
[Style] This function could be below the class to follow [top-down vertical formatting](https://levelup.gitconnected.com/javascript-clean-code-vertical-formatting-d15097f5f30e).

@MaksymilianDemitraszek agreed to do that in the profiler:
https://github.com/software-mansion/protostar/pull/906#discussion_r988748014

Reply:
I wouldn't apply the rules from javascript here. Personally i like this style and unless we have some style guide we want to follow, i'm not really willing to change my mind :|